src/mds/journal.cc

```diff
@@ -1618,13 +1616,17 @@ void EMetaBlob::replay(MDSRank *mds, LogSegment *logseg, MDPeerUpdate *peerup)
 	if (!preallocated_inos.empty())
 	  mds->sessionmap.replay_advance_version();
       }
-      ceph_assert(sessionmapv == mds->sessionmap.get_version());
-    } else {
-      mds->clog->error() << "EMetaBlob.replay sessionmap v " << sessionmapv
-			 << " - " << diff << " > table " << mds->sessionmap.get_version();
-      ceph_assert(g_conf()->mds_wipe_sessions);
-      mds->sessionmap.wipe();
-      mds->sessionmap.set_version(sessionmapv);
+
+      // [repair bad sessionmap updates]
```

Comment:
nit: this really isn't a fix for a bad session update, isn't it?

(sounds misleading imo)

Reply:
Yeah, just copied from `inotable` relevant code. I will fix the comments.